### ISSUE TITLE
Add parallel calculation of the significance

### ISSUE THREAD

**Issue by ThomasKoppensteiner** [Data2Semantics/powerlaws#2]
The significance is currently calculated serially. I think the calculation could be much faster if it is calculated in parallel.

**Comment by pbloem** [comment]
Good idea. Shouldn't take too long to implement. I'll have a look at it when I get a moment.
